script.js

```diff
@@ -1,4 +1,489 @@
+    suit: "♧",
+    rank: 1,
+  },
+  {
+    name: "2",
+    suit: "♧",
+    rank: 2,
+  },
+  {
+    name: "3",
+    suit: "♧",
+    rank: 3,
+  },
+  {
+    name: "4",
+    suit: "♧",
+    rank: 4,
+  },
+  {
+    name: "5",
+    suit: "♧",
+    rank: 5,
+  },
+  {
+    name: "6",
+    suit: "♧",
+    rank: 6,
+  },
+  {
+    name: "7",
+    suit: "♧",
+    rank: 7,
+  },
+  {
+    name: "8",
+    suit: "♧",
+    rank: 8,
+  },
+  {
+    name: "9",
+    suit: "♧",
+    rank: 9,
+  },
+  {
+    name: "10",
+    suit: "♧",
+    rank: 10,
+  },
+  {
+    name: "Jack",
+    suit: "♧",
+    rank: 10,
+  },
+  {
+    name: "Queen",
+    suit: "♧",
+    rank: 10,
+  },
+  {
+    name: "King",
+    suit: "♧",
+    rank: 10,
+  },
+  {
+    name: "ace",
+    suit: "♤",
+    rank: 1,
+  },
+  {
+    name: "2",
+    suit: "♤",
+    rank: 2,
+  },
+  {
+    name: "3",
+    suit: "♤",
+    rank: 3,
+  },
+  {
+    name: "4",
+    suit: "♤",
+    rank: 4,
+  },
+  {
+    name: "5",
+    suit: "♤",
+    rank: 5,
+  },
+  {
+    name: "6",
+    suit: "♤",
+    rank: 6,
+  },
+  {
+    name: "7",
+    suit: "♤",
+    rank: 7,
+  },
+  {
+    name: "8",
+    suit: "♤",
+    rank: 8,
+  },
+  {
+    name: "9",
+    suit: "♤",
+    rank: 9,
+  },
+  {
+    name: "10",
+    suit: "♤",
+    rank: 10,
+  },
+  {
+    name: "Jack",
+    suit: "♤",
+    rank: 10,
+  },
+  {
+    name: "Queen",
+    suit: "♤",
+    rank: 10,
+  },
+  {
+    name: "King",
+    suit: "♤",
+    rank: 10,
+  },
+];
+invalid = `<b>You have enter an invalid input!<br>Please only type 1 or 11!`;
+gameMode = "welcome";
+playerName = "";
+playerBets = "";
+blackjack = 21;
+playingChips = 100;
+p1winCounter = 0;
+pcWinCounter = 0;
+drawCounter = 0;
+p1StartHand = [];
+pcHand = [];
+HOSinvalid = `<b>You have entered an invalid entry ❌!<br>Please only enter hit or stand!<br>Your current hand is ${p1StartHand}<br><br>Hit to have additional cards added to your current hand.<br>Stand if you've burst or<br>Stand to finalize your hand!`;
+blackjackMessage = `<b>BLACKJACK!!<br><br>Your final hand is 21!<br>Computer's turn now.`;
+var shuffle = function () {
+  var shuffledeck = deck[Math.floor(Math.random() * 52)];
+  return shuffledeck;
+};
+
+var randomCard = function () {
+  firstDraw = shuffle();
+  secondDraw = shuffle();
+  pDraw = shuffle();
+  cDraw = shuffle();
+  additionalDraw = shuffle();
+  initialHand = [firstDraw.rank + secondDraw.rank];
+  pciHand = [pDraw.rank + cDraw.rank];
+  if (firstDraw.name == "ace" && secondDraw.name == "ace") {
+    gameMode = "pc";
+    p1StartHand = initialHand;
+    p1StartHand = 21;
+    return blackjackMessage;
+  }
+  if (firstDraw.name == "ace" && secondDraw.rank == 10) {
+    gameMode = "pc";
+    p1StartHand = initialHand;
+    p1StartHand = 21;
+    return blackjackMessage;
+  }
+  if (secondDraw.name == "ace" && firstDraw.rank == 10) {
+    gameMode = "pc";
+    p1StartHand = initialHand;
+    p1StartHand = 21;
+    return blackjackMessage;
+  }
```

Comment:
these are very specific cases, it would be good to think of a way to do this without having to list all the scenarios